src/api/entities/CheckpointSchedule/__tests__/index.ts

```diff
@@ -11,12 +13,14 @@ jest.mock(
 
 describe('CheckpointSchedule class', () => {
   let context: Context;
+  let checkpointSchedule: CheckpointSchedule;
```

Comment:
If you're only using this in a single test, maybe it should be defined in that test